fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/digitalcash/DigitalCashActivity.java

```diff
@@ -39,25 +39,24 @@ protected void onCreate(Bundle savedInstanceState) {
     setupNavigationBar();
     setupBackButton();
 
+    subscribeToSelectItemEvents();
```

Comment:
we subscribe to select/choose item events , or subscribe to the select**ed** item's events ? 
if I understand correctly, the latter :)

Reply:
The latter :)